scipy/sparse/linalg/_isolve/iterative.py

```diff
@@ -113,15 +113,15 @@ def bicg(A, b, x0=None, *, rtol=1e-5, atol=0., maxiter=None, M=None, callback=No
 
     for iteration in range(maxiter):
         if np.linalg.norm(r) < atol:  # Are we done?
-            return postprocess(x), 0
+            return x, 0
 
         z = psolve(r)
         ztilde = rpsolve(rtilde)
         # order matters in this dot product
         rho_cur = dotprod(rtilde, z)
 
         if np.abs(rho_cur) < rhotol:  # Breakdown case
-            return postprocess, -10
```

Comment:
This looks like a bug that was never hit, the intent here clearly was to return `postprocess(x0`